app/start/global.php

```diff
@@ -61,3 +63,9 @@
 App::down(function () {
     return Response::make("Be right back!", 503);
 });
+
+/**
+ * Segment.com tracking.
+ * Since every user is identified under one site, we can "identify" them here.
+ */
+segment_identify(Config::get('app.key'));
```

Comment:
This means we're setting this up on every request. Probably not what we want to do.